modules/recording/recording/recording.tsx

```diff
@@ -204,71 +205,64 @@ const Recording = () => {
               </span>
             </div>
             <div className={recording_styles.controlsContainer}>
-              <button
+              <IconButton
                 onClick={handleDelete}
                 className={recording_styles.actionButton}
+                size="small"
               >
                 <FaTrash size={16} style={{ color: "#DF4949" }} />
-              </button>
-              <button
+              </IconButton>
+
+              <IconButton
```

Comment:
Do we have an icon button?
Why not to have a button with an icon property as component?

IconButton is not bad, will had being my 2nd option